lib/keystores.js

```diff
@@ -128,6 +128,18 @@ api.update = async ({config} = {}) => {
   assert.string(config.controller, 'config.controller');
   assert.number(config.sequence, 'config.sequence');
 
+  const validProps = new Set(['id', 'controller', 'sequence']);
+  for(const key in config) {
+    if(!validProps.has(key)) {
+      throw new BedrockError(
+        'Config can only contain "controller", "sequence" and "id".',
```

Comment:
Oxford commas FTW! :)

```suggestion
        'Config must only contain "controller", "sequence", and "id".',
```